src/libraries/System.Threading.Channels/src/System/Threading/Channels/Channel.cs

```diff
@@ -49,7 +49,22 @@ public static Channel<T> CreateBounded<T>(BoundedChannelOptions options)
                 throw new ArgumentNullException(nameof(options));
             }
 
-            return new BoundedChannel<T>(options.Capacity, options.FullMode, !options.AllowSynchronousContinuations);
+            return new BoundedChannel<T>(options.Capacity, options.FullMode, !options.AllowSynchronousContinuations, itemDropped: null);
+        }
```

Comment:
This method should just delegate to CreateBounded(options, null).

Reply:
Done.